admin-dev/themes/new-theme/js/header.js

```diff
@@ -42,55 +42,57 @@ export default class Header {
     $('.js-quick-link').on('click', (e) => {
       e.preventDefault();
 
-      let method = $(e.target).data('method');
+      const method = $(e.target).data('method');
       let name = null;
 
       if (method === 'add') {
-        let text = $(e.target).data('prompt-text');
-        let link = $(e.target).data('link');
+        const text = $(e.target).data('prompt-text');
+        const link = $(e.target).data('link');
 
         name = prompt(text, link);
       }
-      if (method === 'add' && name || method === 'remove') {
-        let postLink = $(e.target).data('post-link');
-        let quickLinkId = $(e.target).data('quicklink-id');
-        let rand = $(e.target).data('rand');
-        let url = $(e.target).data('url');
-        let icon = $(e.target).data('icon');
+
+      if ((method === 'add' && name) || method === 'remove') {
+        const postLink = $(e.target).data('post-link');
+        const quickLinkId = $(e.target).data('quicklink-id');
+        const rand = $(e.target).data('rand');
+        const url = $(e.target).data('url');
+        const icon = $(e.target).data('icon');
 
         $.ajax({
           type: 'POST',
           headers: {
-            "cache-control": "no-cache"
+            'cache-control': 'no-cache',
           },
           async: true,
           url: `${postLink}&action=GetUrl&rand=${rand}&ajax=1&method=${method}&id_quick_access=${quickLinkId}`,
           data: {
-            "url": url,
-            "name": name,
-            "icon": icon
+            url,
+            name,
+            icon,
           },
-          dataType: "json",
+          dataType: 'json',
           success: (data) => {
-            var quicklink_list = '';
+            let quicklinkList = '';
             $.each(data, (index) => {
-              if (typeof data[index]['name'] !== 'undefined')
-                quicklink_list += '<li><a href="' + data[index]['link'] + '&token=' + data[index]['token'] + '"><i class="icon-chevron-right"></i> ' + data[index]['name'] + '</a></li>';
+              /* eslint-disable-next-line max-len */
+              if (typeof data[index].name !== 'undefined') quicklinkList += `<li><a href="${data[index].link}&token=${data[index].token}"><i class="icon-chevron-right"></i> ${data[index].name}</a></li>`;
             });
 
-            if (typeof data['has_errors'] !== 'undefined' && data['has_errors'])
+            if (typeof data.has_errors !== 'undefined' && data.has_errors) {
```

Comment:
Is it normal to change the access to array : data['has_errors']  => data.has_errors ?

Reply:
yes because in js we can access to objects in different ways but if you don't have `-` or spaces in the attributs name, you must use the dot notation.

Reply:
Ok for me (and thanks ;))